pandas/_libs/parsers.pyx

```diff
@@ -275,14 +276,16 @@ cdef extern from "parser/io.h":
                           size_t *bytes_read, int *status)
 
 
-DEFAULT_CHUNKSIZE = 256 * 1024
-
-
 cdef class TextReader:
     """
 
     # source: StringIO or file object
 
+    ..versionchange:: 1.2.0
+        removed 'compression' argument (outsourced to CParserWrapper).
+        removed 'memory_map' argument: CParserWrapper converts all
+            non-memory-mapped files to file handles. The remaining
+            (string) objects will be opened with 'memory_map=True'.
```

Comment:
Is this implicit behavior okay?

`TextReader` is never called directly (except in tests otherwise it is called through `CParserWrapper`).

Reply:
this is ok